src/assets/styles/constants/colors.ts

```diff
@@ -1,56 +1,59 @@
 export const COLORS = {
-    accent500: "#7367F0",
-    onPrimary: "#FFFFFF",
-    accent100: "#D4D0FA",
-    accent200: "#BFB9F8",
-    accent600: "#695EDA",
-    accent50: "#F1F0FE",
-    accent900: "#302B65",
-    accent700: "#5249AA",
-    primary: "#1A0DAB",
-
-    secondary: "#625B71",
-    onSecondary: "#FFFFFF",
-    secondaryContainer: "#E8DEF8",
-    onSecondaryContainer: "#4A4459",
-    secondaryFixed: "#E8DEF8",
-    secondaryFixedDim: "#1D192B",
-    onSecondaryFixed: "#1D192B",
-    onSecondaryFixedVariant: "#4A4458",
-
-    tertiary: "#7D5260",
-    onTertiary: "#FFFFFF",
-    tertiaryContainer: "#FFD8E4",
-    tertiaryFixed: "#633B48",
-    tertiaryFixedDim: "#EFB8C8",
-    onTertiaryFixed: "#31111D",
-    onTertiaryFixedVariant: "#633B48",
-
-    error: "#B3261E",
-    onError: "#FFFFFF",
-    errorContainer: "#F9DEDC",
-    onErrorContainer: "#852221",
-
-    surfaceDim: "#DED8E1",
-    surface: "#FEF7FF",
-    surfaceBright: "#FEF7FF",
-
-    surfaceContainerLowest: "#FFFFFF",
-    surfaceContainerLow: "#F7F2FA",
-    surfaceContainer: "#F3EDF7",
-    surfaceContainerHigh: "#ECE6F0",
-    surfaceContainerHighest: "#E6E0E9",
-
-    onSurface: "#1D1B20",
-    onSurfaceVariant: "#49454F",
-    outline: "#79747E",
-    outlineVariant: "#CAC4D0",
-
-    inverseSurface: "#322F35",
-    inverseOnSurface: "#F5EFF7",
-    scrim: "#000000",
-    shadow: "#000000",
-
-    disabled: "#D0D5DD",
-    signInContainer: "#F8F7FA"
+  accent500: '#7367F0',
+  onPrimary: '#FFFFFF',
+  accent100: '#D4D0FA',
+  accent200: '#BFB9F8',
+  accent600: '#695EDA',
+  accent50: '#F1F0FE',
+  accent900: '#302B65',
+  accent700: '#5249AA',
+  primary: '#1A0DAB',
+  logo: '#A199F5',
+
+  secondary: '#625B71',
+  onSecondary: '#FFFFFF',
+  secondaryContainer: '#E8DEF8',
+  onSecondaryContainer: '#4A4459',
+  secondaryFixed: '#E8DEF8',
+  secondaryFixedDim: '#1D192B',
+  onSecondaryFixed: '#1D192B',
+  onSecondaryFixedVariant: '#4A4458',
+
+  tertiary: '#7D5260',
+  onTertiary: '#FFFFFF',
+  tertiaryContainer: '#FFD8E4',
+  tertiaryFixed: '#633B48',
+  tertiaryFixedDim: '#EFB8C8',
+  onTertiaryFixed: '#31111D',
+  onTertiaryFixedVariant: '#633B48',
+
+  success: '#008000',
+
+  error: '#B3261E',
+  onError: '#FFFFFF',
+  errorContainer: '#F9DEDC',
+  onErrorContainer: '#852221',
+
+  surfaceDim: '#DED8E1',
+  surface: '#FEF7FF',
+  surfaceBright: '#FEF7FF',
+
+  surfaceContainerLowest: '#FFFFFF',
+  surfaceContainerLow: '#F7F2FA',
+  surfaceContainer: '#F3EDF7',
+  surfaceContainerHigh: '#ECE6F0',
+  surfaceContainerHighest: '#E6E0E9',
+
+  onSurface: '#1D1B20',
+  onSurfaceVariant: '#49454F',
+  outline: '#79747E',
+  outlineVariant: '#CAC4D0',
+
+  inverseSurface: '#322F35',
+  inverseOnSurface: '#F5EFF7',
+  scrim: '#000000',
+  shadow: '#000000',
+
+  disabled: '#D0D5DD',
+  signInContainer: '#F8F7FA',
```

Comment:
I already see this changes in Olexandr PR. Are you doing the same things? I mean eslint rules